### ISSUE TITLE
[Accessibility] key press on legend does not toggle series on IE11

### ISSUE THREAD

**Issue by faisaltaher1** [highcharts/highcharts#10561]
#### Expected behaviour

On Internet Explorer 11, using the accessibility module - after navigating to a legend item via keyboard, corresponding series should be toggled when 'spacebar' or 'enter' key is pressed.

#### Actual behaviour

On Internet Explorer 11, using the accessibility module - after navigating to a legend item via keyboard, corresponding series are not toggled when 'spacebar' or 'enter' key is pressed.

#### Live demo with steps to reproduce
1. Open in IE11 browser: https://www.highcharts.com/samples/highcharts/accessibility/accessible-line/
2. Navigate to legend via keyboard
3. Press 'spacebar' key or 'enter' key to try and toggle
4. Nothing should happen

#### Product version
Highcharts v7.1.1 (and the accessibility module)

#### Affected browser(s)
Internet Explorer 11

**Comment by pawelfus** [comment]
Thanks for reporting @faisaltaher1 ! 

@oysteinmoseng - could you take a look? Confirmed, it works in Chrome (osx), but not in IE11 (windows 10).

**Comment by faisaltaher1** [comment]
No worries, I found a workaround by adding the `useHTML: true` to the legend. With this you can toggle the series in IE11 via keyboard, but I imagine that's not the desired functionality.
```
legend: {
  useHTML: true
}
```

**Comment by raf18seb** [comment]
+1
